README.md

```diff
@@ -61,40 +61,54 @@ These instructions will have the code running on your local or virtual machine.
 
 You will need the following to have a working copy of this software. See
 [installation](#installation-instructions) steps: Note: some of the dependencies will be updated.
-1. Python (>=v3.7.9)
+1. Python (>=v3.13)
 2. [`figshare`](https://github.com/UAL-RE/figshare) - ReDATA's forked copy of [cognoma's figshare](https://github.com/cognoma/figshare)
-3. [`pandas`](https://pandas.pydata.org/) ([1.2.3](https://pandas.pydata.org/pandas-docs/version/1.2.3/))
-4. [`requests`](https://requests.readthedocs.io/en/master/) ([2.22.0](https://requests.readthedocs.io/en/master/2.22.0))
-5. [`numpy`](https://numpy.org/) ([1.20.0](https://numpy.org/devdocs/release/1.20.0-notes.html))
-6. [`jinja2`](https://palletsprojects.com/p/jinja/) ([2.11.2](https://jinja.palletsprojects.com/en/2.11.x/))
-7. [`tabulate`](https://github.com/astanin/python-tabulate) (0.8.3)
-8. [`html2text`](https://pypi.org/project/html2text/) ([2020.1.16](https://pypi.org/project/html2text/2020.1.16/))
+3. [`pandas`](https://pandas.pydata.org/) ([2.2.3](https://pandas.pydata.org/pandas-docs/version/2.2.3/))
+4. [`requests`](https://requests.readthedocs.io/en/master/) ([2.32.0](https://requests.readthedocs.io/en/v2.3.0/))
+5. [`numpy`](https://numpy.org/) ([2.3.5](https://numpy.org/devdocs/release/2.3.5-notes.html))
+6. [`jinja2`](https://palletsprojects.com/p/jinja/) ([3.1.6](https://jinja.palletsprojects.com/en/3.1.x/))
+7. [`tabulate`](https://github.com/astanin/python-tabulate) (0.9.0)
+8. [`html2text`](https://pypi.org/project/html2text/) ([2025.4.15](https://pypi.org/project/html2text/2025.4.15/))
```

Comment:
This doesn't match what's in requirements.txt. Since some dependencies are installed with `ldcoolp-figshare`, recommend updating this section to reflect reality. If you want to call out specific libraries that are being used, that's ok, but the wording should be changed to avoid implications that these libraries need to be installed separately